analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -287,7 +287,8 @@ class Pip(
         // Amend information from "setup.py" with that from "requirements.txt".
         val projectName = when (Pair(setupName.isNotEmpty(), requirementsName.isNotEmpty())) {
             Pair(true, false) -> setupName
-            Pair(false, true) -> requirementsName
+            // As requirementsName cannot be made very unique use the definition file path as projectName:
```

Comment:
This comment refers to code how it was like before this change, and such such might be confusing to read for the developer looking at the current code. I.e., a comment like this rather belongs to the commit message, but you already explained yourself well enough in there I believe. Here, I'd prefer to read something like:

    // In case of only a requirements file without further meta-data, use the relative path to the analyzer root as a unique project name.

Reply:
done.